client/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "client",
   "version": "0.1.0",
-  "proxy": "http://localhost:3001/",
+  "proxy": "http://localhost:3000",
```

Comment:
Is this related to the issue you were having? Is there a specific reason we're changing it to port 3001?

Reply:
We're still going to continue using 3000. It's just that for my machine, I need to use 3001 until I figure out what is using 3000

Reply:
Ah gotcha.